src/mrack/utils.py

```diff
@@ -153,7 +153,8 @@ def get_host_from_metadata(metadata, name):
     domains = metadata.get("domains", [])
     for domain in domains:
         for host in domain.get("hosts", []):
-            if host["name"] == name:
+            # use startswith as for ad we use only first part of FQDN
+            if host["name"].startswith(name):
```

Comment:
Where is this change needed?

Reply:
We are passing `host.name` as `name` parameter which is `Host` object attribute `name` and this is set based on response from server.
If we ask for name `ad.domain.com` the `host.name` will be `ad` as change is we ask for this name in OpenStack.